<ui version="4.0" >
 <class>VdmsAddClass</class>
 <widget class="QDialog" name="VdmsAddClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>450</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Asistente del VdmSound</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/grlida.qrc" >
    <normaloff>:/img16/vdmsound.png</normaloff>:/img16/vdmsound.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="imgLogoSvm" >
     <property name="minimumSize" >
      <size>
       <width>450</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../res/grlida.qrc" >:/images/wizard_vdmsound.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="lb_Vdms_3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Parametros:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_Vdms_4" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Dirección del Icono:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_Icono" >
      <property name="geometry" >
       <rect>
        <x>380</x>
        <y>170</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_ExeJuego" >
      <property name="geometry" >
       <rect>
        <x>380</x>
        <y>110</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_FileConfg" >
      <property name="geometry" >
       <rect>
        <x>380</x>
        <y>60</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lb_Vdms_1" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Nombre del Archivo de configuración a Guardar ejemplo: Nombre_Archivo.vlp</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_Datos_1" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Titulo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDatos_Titulo" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>10</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_Vdms_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>90</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Ejecutable (exe,com,bat) del Juego:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtVdms_params" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>140</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtVdms_path_conf" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtVdms_path_exe" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>110</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtVdms_icon" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>170</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_Icono_clear" >
      <property name="geometry" >
       <rect>
        <x>414</x>
        <y>170</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize" >
       <size>
        <width>30</width>
        <height>26</height>
       </size>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_FileConfg_clear" >
      <property name="geometry" >
       <rect>
        <x>414</x>
        <y>60</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize" >
       <size>
        <width>30</width>
        <height>26</height>
       </size>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVdms_ExeJuego_clear" >
      <property name="geometry" >
       <rect>
        <x>414</x>
        <y>110</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize" >
       <size>
        <width>30</width>
        <height>26</height>
       </size>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDescargarInfo" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>260</y>
        <width>162</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text" >
       <string>Descargar la Info del Juego</string>
      </property>
      <property name="icon" >
       <iconset resource="../res/grlida.qrc" >
        <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>97</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>97</width>
         <height>25</height>
        </size>
       </property>
       <property name="text" >
        <string>Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/grlida.qrc" >
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>97</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>97</width>
         <height>25</height>
        </size>
       </property>
       <property name="text" >
        <string>Aceptar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/grlida.qrc" >
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtDatos_Titulo</tabstop>
  <tabstop>txtVdms_path_conf</tabstop>
  <tabstop>btnVdms_FileConfg</tabstop>
  <tabstop>btnVdms_FileConfg_clear</tabstop>
  <tabstop>txtVdms_path_exe</tabstop>
  <tabstop>btnVdms_ExeJuego</tabstop>
  <tabstop>btnVdms_ExeJuego_clear</tabstop>
  <tabstop>txtVdms_params</tabstop>
  <tabstop>txtVdms_icon</tabstop>
  <tabstop>btnVdms_Icono</tabstop>
  <tabstop>btnVdms_Icono_clear</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/grlida.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>VdmsAddClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>73</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel" >
     <x>214</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnVdms_FileConfg_clear</sender>
   <signal>clicked()</signal>
   <receiver>txtVdms_path_conf</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>423</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel" >
     <x>336</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnVdms_ExeJuego_clear</sender>
   <signal>clicked()</signal>
   <receiver>txtVdms_path_exe</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>423</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel" >
     <x>364</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnVdms_Icono_clear</sender>
   <signal>clicked()</signal>
   <receiver>txtVdms_icon</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>428</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel" >
     <x>354</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
